R/PythonPkgWrapperUtils.R

```diff
@@ -0,0 +1,443 @@
+# ------------------------------------------------------------------------------
+#
+#   Helpers for wrapping python packages
+#
+# ------------------------------------------------------------------------------
+
+#' Determines args and kwargs
+#'
+#' This function determines args and kwargs for a given list of values.
```

Comment:
This function takes the list of arguments passed to an R function and groups them into the (1) unnamed / positional arguments and the (2) the named / keyword arguments to pass to the corresponding Python function.